server/modules/selva/module/inherit.c

```diff
@@ -218,15 +220,18 @@ int SelvaInheritCommand(RedisModuleCtx *ctx, RedisModuleString **argv, int argc)
     RedisModule_AutoMemory(ctx);
     int err;
 
-    const int ARGV_REDIS_KEY     = 1;
-    const int ARGV_NODE_ID       = 2;
-    const int ARGV_TYPES         = 3;
-    const int ARGV_FIELD_NAMES   = 4;
+    const int ARGV_LANG          = 1;
```

Comment:
Actually the redis API requires that the first argument should be a key in redis. Not sure if it really matters 🤷‍♂️